lib/interface/login/login.dart

```diff
@@ -80,19 +91,24 @@ class LoginScreenState extends State<LoginScreen>
   @override
   Widget build(BuildContext context) {
     _ctx = context;
-    var loginBtn = new RaisedButton(
-      onPressed: _submit,
-      child: new Text(MyLocalizations.of(context, 'LOGIN')),
-      color: Colors.primaries[0],
+    var loginBtn = ButtonTheme(
+      minWidth: MediaQuery.of(context).size.width * 0.5,
+      child: new RaisedButton(
+        onPressed: _submit,
+        child: new Text(MyLocalizations.of(context, 'LOGIN')),
+        color: Colors.lightBlue,
+      ),
     );
-    var loginForm = new Column(
+    var loginForm = Column(
+      mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
Try not to use alignment where a widget can be imported elsewhere.
Think of every widget as a complete page.